include/roaring/roaring.h

```diff
@@ -335,6 +335,13 @@ bool roaring_bitmap_contains_range(const roaring_bitmap_t *r,
                                    uint64_t range_start,
                                    uint64_t range_end);
 
+/**
+ * Check multiple values sized by n_args return the existence of
+ * each element at the index by results.
+ */
```

Comment:
You probably need to tell us something about the parameters. E.g., I suppose that results must have n_args elements?